schemas/dev.amp.snowplow/amp_page_ping/jsonschema/1-0-0

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "dev.amp.snowplow",
+    "name": "amp_page_ping",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Schema for amp-specific page ping events",
+  "properties": {
+    "scrollLeft": {
+      "type": "integer",
+      "description": "Pixels scrolled from the left, as provided by AMP"
+    },
+    "scrollWidth": {
+      "type": "integer",
+      "description": "Width of page in pixels, as provided by AMP",
+      "minimum": 0
+    },
+    "viewportWidth": {
+      "type": "integer",
+      "description": "Width of viewport in pixels, taking into account zoom and window size, as provided by AMP",
+      "minimum": 0
+    },
+    "scrollTop": {
+      "type": "integer",
+      "description": "Pixels scrolled from the top"
```

Comment:
Can it be negative?

Reply:
Probably no but possibly yes. This is why I've not set a minimum for this one.